docs/source/notebooks/mmm/mmm_case_study.ipynb

```diff
@@ -45,7 +45,7 @@
   },
```

Comment:
<p>Can we change the verbiage here as well. The use of "reasonable" is a bit strange </p><p> </p><p>Also, would we want to comment on the fact that the prior predictive includes negative values... That might element to highlight</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/1170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>